docs/reviewing-tests/index.md

```diff
@@ -38,3 +38,24 @@ working on other things as tests frequently only get pushed upstream
 once the code lands in their implementation.
 
 To assist with test reviews, a [review checklist](checklist) is available.
+
+## Notifications
+
+META.yml files are used only to indicate who should be notified of pull
+requests.  If you are interested in receiving notifications of proposed
+changes to tests in a given directory, feel free to add yourself to the
+META.yml file. Anyone with expertise in the specification under test can
```

Comment:
"Notifications" is a better header for the first part of this paragraph, but not at all for the second part. Split the paragraph before "Anyone" and add a new "Review" header? Then maybe the "Finding contributions to review" header can go away again.

Reply:
Sounds good!